src/lib/common/ConfirmationModal.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+    import Modal from "./Modal.svelte";
+    import { createEventDispatcher } from "svelte";
+    const dispatch = createEventDispatcher();
```

Comment:
```suggestion

    const dispatch = createEventDispatcher();
```

Tiny nitpick, space between imports and other stuff, for consistency. I wish prettier would enforce stuff like this :\